setup.py

```diff
@@ -51,7 +51,7 @@
     'retry_decorator>=1.0.0',
     'six>=1.16.0',
     # aiohttp is the extra dependency that contains requests lib.
-    'google-auth[aiohttp]>=2.5.0',
+    'google-auth[aiohttp]==2.17.0',
```

Comment:
If we don't keep this, then pip would install the latest version of google-auth, due to which there's mismatch in the response, and release test would fail.